app/views/welcome_mailer/notify_newcomer.html.erb

```diff
@@ -18,6 +18,13 @@
 
     Looking forward to hearing from you soon,
 
+    Please click on the link below to verify your email.
+
+    <% verifaction_link="#{ActionMailer::Base.default_url_options[:host]}/verify/#{@user.generate_token}" %>
+    <%= link_to "#{verifaction_link}", verifaction_link%>
+    
+    In case you are not able to click the link, try copy pasting the link in the browser.
+
```

Comment:
Hi @jywarren make sure too copy this change to the **welcome-email-body** feature whenever you push this PR to production as the changes made here are just on the fallback email body and the production would send the email using the welcome-email-body feature's content(if it's present). 

Thanks!

Reply:
Ah, then this is an issue - we can't execute code in the features system, so this will need to be moved outside the `<% end %>` on this line: https://github.com/publiclab/plots2/pull/3992/files/b74bbcbd3d85ff70b18bc7eae14bf79105270976#diff-a3214821bc5b12012e6fbb54ad82f05aR29

Reply:
@jywarren . Sorry but I am not getting what we are supposed to do?

Reply:
Oh I think I got it. We need to move the verification link and the corresponding instructions regarding the mail outside the if else block right?